MqttClient_Hello/app/application.cpp

```diff
@@ -38,6 +38,7 @@ void onMessageReceived(String topic, String message)
 // Run MQTT client
 void startMqttClient()
 {
+	mqtt.setWill("last/will","The connection from this device is lost:(", 1, true);
```

Comment:
@Tavalin Can you please explain what exactly does not look ok in the line above? The method mqtt.setWill is called with 4 parameters, 2 strings, one int with value 1 and one boolean with value true.